lms/static/styles/_base.scss

```diff
@@ -0,0 +1,17 @@
+// Enable Tailwind base layer. Apply CSS normalize & reset (Tailwind preflight)
+@use 'tailwindcss/base';
+
+// Element styles
+@layer base {
+  html,
+  body {
+    height: 100%;
+  }
+
+  // Base typography styling
+  body {
+    @apply font-sans;
+    @apply text-tiny;
+    @apply text-grey-9;
+  }
+}
```

Comment:
A rule has been removed from base styles here that set `cursor: pointer` for buttons. This is handled by CSS normalization now.

As an aside, Tailwind preflight automatically applies the default `sans` font-family to the `html` element as part of preflight, but I think it's advisable to be really clear about the base font we're setting up here and not have any part of it be magical.

I'm still ruminating on color names, which is why we have `text-grey-9` here instead of an abstracted color name that means "default body text color."

Our default body font size for LMS _is_ tiny. It's a custom size in our local Tailwind config.